exercises/protein-translation/canonical-data.json

```diff
@@ -157,14 +198,42 @@
           },
           "expected": []
         },
+        {
+          "uuid": "f4d9d8ee-00a8-47bf-a1e3-1641d4428e54",
+          "description": "Sequence of two protein codons translates into proteins",
+          "property": "proteins",
+          "input": {
+            "strand": "UUUUUU"
+          },
+          "expected": [
+            "Phenylalanine",
+            "Phenylalanine"
+          ]
+        },
+        {
+          "uuid": "dd22eef3-b4f1-4ad6-bb0b-27093c090a9d",
+          "description": "Sequence of two different protein codons translates into proteins",
+          "property": "proteins",
+          "input": {
+            "strand": "UUAUUG"
+          },
+          "expected": [
+            "Leucine",
+            "Leucine"
```

Comment:
Shouldn't these be different?

Reply:
They should not but the name isn't clear.

Reply:
(The _codon_ is different, the _protein_ is not)